frontend/config-assistant/src/components/toolbar/fetchExampleSchemas.ts

```diff
@@ -0,0 +1,26 @@
+import {schemaCollection} from '@/data/SchemaCollection';
+import {ChangeResponsible, useSessionStore} from '@/store/sessionStore';
+import {useDataStore} from '@/store/dataStore';
+import {newEmptyFile} from '@/components/toolbar/clearFile';
+import {errorService} from '@/main';
+
+export async function fetchExampleSchema(schemaKey: string, toast?: any): Promise<void> {
```

Comment:
if the function fetches just one example schema, I recommend renaming the file to using singular "schema" instead of "schemas" also

Reply:
in this function fetches multiple example schemas , not only one animal schema, default schema and 2 more I guess @Logende